src/components/player/YoutubePlayer.tsx

```diff
@@ -70,7 +70,7 @@ export function usePlayer(player?: YouTubePlayer) {
           setVolume(await player.getVolume());
           setMuted(await player.isMuted());
         }
-      }, 333);
+      }, 200);
```

Comment:
Is this needed change?

Reply:
Volume change event from keyboard shortcut is limited to the update interval of the `YoutubePlayer` component.
So, I thought update interval should be accelerated.
(but yeah it's a logic problem. I'll think better solution.)

Reply:
maybe if you change it directly using player, without waiting for react states to update. Something like

```js
player.setVolume(await player.getVolume() + 5)
```

The visual will lag behind by 333ms, but that's ok

Reply:
Thanks for the advice. I'll change to it.